database/dbPersons.php

```diff
@@ -182,6 +182,67 @@ function update_volunteer_hours($eventname, $username, $new_start_time, $new_end
 	return $result;
 }
 
+/*@@@ Thomas */
+
+/* Check-in a user by adding a new row and with start_time to dbpersonhours */
+function check_in($personID, $eventID, $start_time) {
+    $con=connect();
+	$query = "INSERT INTO dbpersonhours (personID, eventID, start_time) VALUES ( '" .$personID. "', '" .$eventID. "', '" .$start_time. "')";
+    //$query = 'UPDATE dbpersonhours SET start_time = "' . $new_start_time . '", end_time = "' . $new_end_time . ' WHERE eventID = "' . $eventid . '" AND personID = "' . $username . '"';
+	$result = mysqli_query($con,$query);
+	mysqli_close($con);
+	return $result;
+}
+
+/* Check-out a user by updating their end_time */
+function check_out($personID, $eventID, $end_time) {
+    $con=connect();
+    $query = "UPDATE dbpersonhours SET end_time = '" . $end_time . "' WHERE eventID = '" .$eventID. "' AND personID = '" .$personID. "' and end_time IS NULL";
+	$result = mysqli_query($con,$query);
+	mysqli_close($con);
```

Comment:
indentation does not match rest of function

Reply:
Addressed above.